examples/style_transfer/model/README.md

```diff
@@ -0,0 +1,79 @@
+## Training and Conversion
```

Comment:
@miaobin please upload trained/converted models to google drive as well with shared permission.